lib/utils/siprec-utils.js

```diff
@@ -48,8 +48,17 @@ const parseSiprecPayload = (req, logger) => {
     }
 
     if (!sdp || !meta) {
-      logger.info({payload: req.payload}, 'invalid SIPREC payload');
-      return reject(new Error('expected multipart SIPREC body'));
+      if (!meta && sdp) {
```

Comment:
nesting this in the if statement above makes it difficult to follow.  can you just have if..else if else